api/src/main/java/io/strimzi/api/kafka/model/connect/build/DownloadableArtifact.java

```diff
@@ -30,8 +29,8 @@
     @Description("URL of the artifact which will be downloaded. " +
             "Strimzi does not do any security scanning of the downloaded artifacts. " +
             "For security reasons, you should first verify the artifacts manually and configure the checksum verification to make sure the same artifact is used in the automated build. " +
-            "Required.")
-    @JsonProperty(required = true)
+            "Required." +
+            "Not applicable to the `maven` artifact type.")
```

Comment:
Missing spaces? + is it really `Required` if it is not applicable to ...? Maybe you should try to reweord it something like this?

```suggestion
            "Required for `jar`, `zip`, and `tgz` artifacts. " +
            "Not applicable to the `maven` artifact type.")
```